### PR TITLE
Scythe rulebook

### PR DESCRIPTION
**Pull request by andrewmassengale**
Adding Scythe rulebook and updating corresponding homepage.yml file.

### CODE REVIEW

File: rulebooks/scythe.md [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+title: Scythe
+source: https://www.orderofgamers.com/downloads/Scythe_v1.7.pdf
+tags:
+  - area-control
+  - variable-player-powers
+  - grid-movement
+  - simultaneous-action-selection
+---
+
+# SETUP
```
Comment (camden) [inline rulebooks/scythe.md]: Can we make these headers title case?

File: rulebooks/scythe.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+
+The player with the lowest number in the label on their player mat goes first; play proceeds clockwise.
+
+# GAMEPLAY
+
+On your turn, do the following in order:
+
+1. Place your action token** on a different section of your player mat than where it was on your previous turn.
+
+2. Take the top-row action** on that section **(optional)**.
```
Comment (camden) [inline rulebooks/scythe.md]: These double asterisks (after "action") are matching with the first pair before "(optional)", leaving the last two asterisks visible in the final rulebook.

File: rulebooks/scythe.md [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+
+You may not use a Move action to move any unit from the board into any home base (including yours).
+
+**If you move your character into a territory with an Encounter token,** their movement ends and they cannot move again this turn. After resolving all combats, if your character is still there, discard the token and resolve the encounter.
+
+There is no limit to the number of same-faction units that can be in a territory.
+
+### BOLSTER
+
+Pay the $1 cost and gain one of the following: ```
```
Comment (camden) [inline rulebooks/scythe.md]: Can we remove these ```?

File: rulebooks/scythe.md (hunk 1 above)
Comment (andrewmassengale) [reply rulebooks/scythe.md]: This is done.

File: rulebooks/scythe.md (hunk 2 above)
Comment (andrewmassengale) [reply rulebooks/scythe.md]: This has been fixed.

File: rulebooks/scythe.md (hunk 3 above)
Comment (andrewmassengale) [reply rulebooks/scythe.md]: These tick marks have been removed.

**Comment by camden** [comment]
Thanks!!

**Pull request merged**